### PR TITLE
ci: run the integration tests on more distributions

### PR DESCRIPTION
**Pull request by haraldh**
Add OpenSuse and Arch Linux.
Only run extended tests on `Fedora:latest` to reduce the workload.

Also remove the `hung_task_panic=1` on the kernel command line, as qemu
tends to be really slow in non-kvm mode.

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        runs-on: ubuntu-latest
+        timeout-minutes: 45
+        concurrency:
+            group: ${{ github.workflow }}-${{ github.ref }}-${{ matrix.container }}-${{ matrix.test }}
+            cancel-in-progress: true
+        strategy:
+            matrix:
+                container: [
+                        "fedora:latest",
+                        "opensuse:latest",
```
Comment (danimo) [inline .github/workflows/integration.yml:47]: `opensuse:latest` has been deprecated for quite a while now noted in the description. I am not even sure what state this is. We should be using `opensuse/tumbleweed:latest`  and probably add `opensuse/leap:latest`, too.

**Review by danimo — CHANGES_REQUESTED** [review]

File: .github/workflows/integration.yml (hunk 1 above)
Comment (haraldh) [reply .github/workflows/integration.yml:47]: it is in fact opensuse/tumbleweed:latest

Comment (haraldh) [reply .github/workflows/integration.yml:47]: https://github.com/dracutdevs/dracut/blob/5a04bd2dd3a091af333c534f4534c87c5175fe13/test/container/Dockerfile-OpenSuse-latest#L1

Comment (danimo) [reply .github/workflows/integration.yml:47]: my bad, thanks!

**Review by danimo — APPROVED** [review]

**Review by johannbg — APPROVED** [review]

**Pull request merged**